pkg/controller/shared/nodedeployment-migration/api/types.go

```diff
@@ -84,7 +84,6 @@ type OperatingSystemSpec struct {
 	Ubuntu      *UbuntuSpec      `json:"ubuntu,omitempty"`
 	AmazonLinux *AmazonLinuxSpec `json:"amzn2,omitempty"`
 	CentOS      *CentOSSpec      `json:"centos,omitempty"`
-	SLES        *SLESSpec        `json:"sles,omitempty"`
```

Comment:
This might cause issues when decoding existing InitialMD's. Maybe we should keep the field here, but simply not use it during conversion?

Reply:
Makes sense, I have reverted this change.